css/style.css

```diff
@@ -0,0 +1,189 @@
+
+.body {
+  max-width: 1440px;
+  background: white;
+  margin: 0 60px 24px 42.72px;
+  font-family: "Montserrat";
+  color: #000000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)